tools/topology/topology2/platform/intel/alh-2nd-spk.conf

```diff
@@ -1,32 +0,0 @@
-
```

Comment:
Would be helpful for mention the dependency to alsa-utils. E.g. https://github.com/alsa-project/alsa-utils/pull/187 and/or alsa-utils version (which is harder as no released version has this).